src/scripts/scrumHelper.js

```diff
@@ -113,21 +116,24 @@ function allIncluded() {
 			'-' +
 			('00' + lastWeekMonth.toString()).slice(-2) +
 			'-' +
-			('00' + lastWeekDay.toString()).slice(-2);
+			('00' + lastWeekDay.toString()).slice(-2) +
+			'T00:00:00Z';
 		return lastWeekDisplayPadded;
 	}
 	function getToday() {
 		var today = new Date();
-		var Week = new Date(today.getFullYear(), today.getMonth(), today.getDate());
-		var WeekMonth = Week.getMonth() + 1;
-		var WeekDay = Week.getDate();
-		var WeekYear = Week.getFullYear();
+		today.setHours(23, 59, 59, 999);
+		// var Week = new Date(today.getFullYear(), today.getMonth(), today.getDate());
+		var WeekMonth = today.getMonth() + 1;
```

Comment:
Please update the variables name to make them follow the same style (camelCase).